go-acme/lego
# Add DNS provider for Combell (reseller only)

Closes #1573

@jwaes can you run the command with your domain, email, credentials, etc. (and put the output of this command to a comment)?


```bash
make build
rm -rf .lego

COMBELL_API_KEY="xxx" \
COMBELL_API_SECRET="yyy" \
./dist/lego run --dns combell -d '*.example.com' -d example.com -s letsencrypt-staging
```

Note that the wildcard domain is important.

<details><summary>How to test this PR?</summary>

1. You need [Go](https://go.dev/doc/install)
2. Check out the PR:
    ```bash
    git clone https://github.com/ldez/lego.git
    cd lego
    git checkout ldez:feat/combell
    ```
3. Compile lego:
    - if you have `make`: `make build`
    - if you don't have `make`: `go build -o dist/lego ./cmd/lego`
4. Run the following command with your information (email, domain, credentials):
    ```bash
    COMBELL_API_KEY="xxx" \
    COMBELL_API_SECRET="yyy" \
    ./dist/lego run --dns combell -d '*.example.com' -d example.com -s letsencrypt-staging
    ```
   **The wildcard domain is important**
5. Before each run of the command, you should clean your local environment:
    ```bash
    rm -rf .lego
    ```

</details>
